app/src/main/java/org/simple/clinic/summary/teleconsultation/contactdoctor/DoctorListItem.kt

```diff
@@ -0,0 +1,69 @@
+package org.simple.clinic.summary.teleconsultation.contactdoctor
+
+import androidx.recyclerview.widget.DiffUtil
+import io.reactivex.subjects.Subject
+import kotlinx.android.synthetic.main.list_contact_doctor.*
+import org.simple.clinic.R
+import org.simple.clinic.summary.teleconsultation.contactdoctor.DoctorListItem.Event.ContactMode.SMS
+import org.simple.clinic.summary.teleconsultation.contactdoctor.DoctorListItem.Event.ContactMode.WhatsApp
+import org.simple.clinic.summary.teleconsultation.sync.MedicalOfficer
+import org.simple.clinic.widgets.ItemAdapter
+import org.simple.clinic.widgets.recyclerview.ViewHolderX
+
+data class DoctorListItem(
+    private val name: String,
+    private val number: String
+) : ItemAdapter.Item<DoctorListItem.Event> {
+
+  companion object {
+    fun from(medicalOfficers: List<MedicalOfficer>): List<DoctorListItem> {
+      return medicalOfficers
+          .map {
+            DoctorListItem(
+                name = it.fullName,
+                number = it.phoneNumber
+            )
+          }
+    }
+  }
+
+  class DiffCallback : DiffUtil.ItemCallback<DoctorListItem>() {
+    override fun areItemsTheSame(oldItem: DoctorListItem, newItem: DoctorListItem): Boolean {
+      return oldItem.number == newItem.number
+    }
+
+    override fun areContentsTheSame(oldItem: DoctorListItem, newItem: DoctorListItem): Boolean {
+      return oldItem == newItem
+    }
+  }
+
+  override fun layoutResId(): Int = R.layout.list_contact_doctor
+
+  override fun render(holder: ViewHolderX, subject: Subject<Event>) {
+    holder.doctorNameTextView.text = name
+    holder.doctorNumberTextView.text = number
+
+    holder.whatsAppDoctorImageView.setOnClickListener {
+      subject.onNext(Event.MessageDoctorClicked(
+          phoneNumber = number,
+          contactMode = WhatsApp
+      ))
+    }
+    holder.messageDoctorImageView.setOnClickListener {
+      subject.onNext(Event.MessageDoctorClicked(
+          phoneNumber = number,
+          contactMode = SMS
+      ))
+    }
+  }
+
+  sealed class Event {
+
+    enum class ContactMode {
+      WhatsApp,
+      SMS
+    }
```

Comment:
We could also have two separate click events instead of an enum. This is merely a suggestion, no need to implement it.

Reply:
Yes, it's just a temporary implementation for now. I will implement click events later on